Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN chown -R app .
 
 USER app
 
-ENV JAVA_OPTS="-XX:+UseParNewGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10"
```

Comment:
did you mean to remove the existing opts or just undo your proposed change to them?